src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -198,7 +198,7 @@
 examples:
   - name: Remove a network rule. (autogenerated)
     text: |
-        az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet MySubnetID
+        az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet mysubnet
```

Comment:
The value for --subnet is ID or name of subnet. In this way, previous one is also acceptable.

Reply:
Fixed